OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -64,8 +64,12 @@ public Attack(Actor self, Target target, bool allowMovement, bool forceAttack, C
 			    || target.Type == TargetType.FrozenActor || target.Type == TargetType.Terrain)
 			{
 				lastVisibleTarget = Target.FromPos(target.CenterPosition);
-				lastVisibleMaximumRange = attackTraits.Where(x => !x.IsTraitDisabled)
-					.Min(x => x.GetMaximumRangeVersusTarget(target));
+				var activeAttackTraits = attackTraits.Where(x => !x.IsTraitDisabled);
+				if (!activeAttackTraits.Any())
```

Comment:
Something would have to be quite bogus for this to be false here in the ctor. Crashing is the correct behaviour here.

Reply:
Removed the check in the constructor.